### ISSUE TITLE
dynamodb streams region inconsistent

### ISSUE THREAD

**Issue by FarhadF** [localstack/localstack#1513]
```
aws --endpoint=http://localhost:4569 dynamodb describe-table --table-name mytable
{
    "Table": {
...
        "LatestStreamArn": "arn:aws:dynamodb:us-east-1:000000000000:table/mytable/stream/2019-08-29T09:30:07.758"
    }
}
```
```
aws --endpoint=http://localhost:4569 dynamodbstreams list-streams
{
            "TableName": "mytable", 
            "StreamArn": "arn:aws:dynamodb:ddblocal:000000000000:table/mytable/stream/2019-08-29T09:30:07.758", 
            "StreamLabel": "2019-08-29T09:30:07.758"
        }
```

Please note last stream arn from first to streamarn from the second, the region is us-east-1 in first and ddblocal in second

**Comment by rejdzu** [comment]
I have the same or similar issue. Describe stream method is not working on both dynamodb and dynamodbstream ports. I've tried streamarn from describe table and list-stream commands. They either return 500 or resource not found exception

This is exception from docker
Traceback (most recent call last):
localstack_1  |   File "/opt/code/localstack/.venv/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app                                                                                                                                                                                                                                                       
localstack_1  |     response = self.full_dispatch_request()                                                                                                                                                                                                                                                                                                                
localstack_1  |   File "/opt/code/localstack/.venv/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request                                                                                                                                                                                                                                          
localstack_1  |     rv = self.handle_user_exception(e)                                                                                                                                                                                                                                                                                                                     
localstack_1  |   File "/opt/code/localstack/.venv/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
localstack_1  |     return cors_after_request(app.make_response(f(*args, **kwargs)))                                                                                                                                                                                                                                                                                       
localstack_1  |   File "/opt/code/localstack/.venv/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception                                                                                                                                                                                                                                          
localstack_1  |     reraise(exc_type, exc_value, tb)                                                                                                                                                                                                                                                                                                                       
localstack_1  |   File "/opt/code/localstack/.venv/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise                                                                                                                                                                                                                                                      
localstack_1  |     raise value
localstack_1  |   File "/opt/code/localstack/.venv/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request                                                                                                                                                                                                                                          
localstack_1  |     rv = self.dispatch_request()                                                                                                                                                                                                                                                                                                                           
localstack_1  |   File "/opt/code/localstack/.venv/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request                                                                                                                                                                                                                                               
localstack_1  |     return self.view_functions[rule.endpoint](**req.view_args)                                                                                                                                                                                                                                                                                             
localstack_1  |   File "/opt/code/localstack/localstack/services/dynamodbstreams/dynamodbstreams_api.py", line 83, in post_request                                                                                                                                                                                                                                         
localstack_1  |     shard['ShardId'] = shard_id(stream_name, shard['ShardId'])
localstack_1  |   File "/opt/code/localstack/localstack/services/dynamodbstreams/dynamodbstreams_api.py", line 141, in shard_id                                                                                                                                                                                                                                            
localstack_1  |     return '-'.join([kinesis_shard_id, random_id(stream_arn, kinesis_shard_id)])                                                                                                                                                                                                                                                                          
localstack_1  |   File "/opt/code/localstack/localstack/services/dynamodbstreams/dynamodbstreams_api.py", line 137, in random_id                                                                                                                                                                                                                                           
localstack_1  |     return uuid.uuid5(namespace, to_bytes(kinesis_shard_id)).hex                                                                                                                                                                                                                                                                                           
localstack_1  |   File "/usr/lib/python3.6/uuid.py", line 628, in uuid5                                                                                                                                                                                                                                                                                                    
localstack_1  |     hash = sha1(namespace.bytes + bytes(name, "utf-8")).digest()                                                                                                                                                                                                                                                                                           
localstack_1  | TypeError: encoding without a string argument

**Comment by whummer** [comment]
Fixed in #1518
